diesel_cli/src/print_schema.rs

```diff
@@ -118,6 +134,20 @@ impl<'a> Display for ColumnDefinitions<'a> {
     }
 }
 
+struct Joinable<'a>(&'a ForeignKeyConstraint);
+
+impl<'a> Display for Joinable<'a> {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        writeln!(
+            f,
+            "joinable!({} -> {} ({}));",
```

Comment:
I see you're using rustfmt style now. I'd put the first two args on the first line, but whatever